### PR TITLE
fix: add android and ios folders

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR fixes the `android` and `ios` folders not being installed when running the template. 

Changes:
- add `android` and `ios` folders to `/template`
- update `.gitignore` to ignore `ios/Pods`

However, I am only getting the whole template to work when using the flag `--ignore-existing`

so this works:
`npx react-native init TestApp --template=https://github.com/echobind/react-native-template#jsjoeio/add-ios-andorid-versions`

I'm not sure if that is a blocker for this PR but worth-noting

### CODE REVIEW

**Comment by cball** [comment]
@jsjoeio confirmed working on ios without `--ignore-existing`. Checking android.

File: .gitignore [hunk 1]
```diff
@@ -39,7 +39,6 @@ yarn-error.log
 # BUCK
 buck-out/
 \.buckd/
-*.keystore
```
Comment (cball) [inline .gitignore]: We should do this as:

```
*.keystore
!debug.keystore
```
That would avoid all keystores except debug.keystore

Comment (jsjoeio) [reply .gitignore]: I was wondering if there was a better way - good thinking! I'll make that change now.

Comment (jsjoeio) [reply .gitignore]: made the change, going to squash and merge!

**Pull request merged**